packages/forms-web-app/src/controllers/selected-appeal/questionnaire-details/index.js

```diff
@@ -34,7 +35,8 @@ exports.get = (layoutTemplate = 'layouts/no-banner-link/main.njk') => {
 			throw new Error('Unknown role');
 		}
 
-		const userEmail = userType === LPA_USER_ROLE ? req.session.lpaUser?.email : req.session.email;
+		const lpaUser = getUserFromSession(req);
+		const userEmail = userType === LPA_USER_ROLE ? lpaUser?.email : req.session.email;
```

Comment:
is it worth putting this in the service as a function?

Reply:
this should be in the id token, so can probably move to using that going forward, can look at in another PR as this one is pretty chunky